src/ampJson.js

```diff
@@ -1,7 +1,14 @@
-export default function ampJson(data, component) {
+export default function ampJson(data, component, attributes={}) {
   if (!data || !component) return '';
+
+  const attributesString = attributes ?
+    Object.entries(attributes).map(([key, val]) => (
+      `${key}=${val}`
+    )).join(' ').trim() :
+    undefined;
```

Comment:
- Does `val` need to be wrapped in quotes? ie: key="val"
- `undefined` is weird here because then it will be the string `"undefined"`